modules/perception/lidar/lib/detection/lidar_point_pillars/include/point_pillars_detection.h

```diff
@@ -42,6 +42,8 @@ class PointPillarsDetection {
 
   bool Detect(const DetectionOptions& options, LidarFrame* frame);
 
+  std::string Name() { return "PointPillarsDetection"; }
```

Comment:
Use a static const variable or a gflag for the string "PointPillarsDetection".

Reply:
I think so, avoid return a string var directly.

Reply:
Have made it const.